ui/App/ProjectScreen/Source/Commit.svelte

```diff
@@ -153,6 +192,38 @@
       </div>
     </div>
 
+    {#each anchors as anchor}
+      <div class="anchor">
+        <div class="header">
+          <div style="display: flex;">
+            <Icon.Anchor />
+            <p
+              class="typo-text-bold"
+              style="color: var(--color-foreground-level-6); margin: 0 0.5rem 0 0.5rem;">
+              Anchored by
+            </p>
```

Comment:
See my other comment about making this a `<span>`.

Reply:
15487c135165f2a826a5ff4a1337d7ebfc5f2f8f